pages/common/bb.md

```diff
@@ -0,0 +1,20 @@
+# bb
+
+> Native Clojure interpreter for scripting.
+> More information: <https://book.babashka.org/#usage>.
+
+- [e]valuate an expression:
+
+`bb -e "(+ 1 2 3)"`
+
+- Evaluate a script [f]ile:
+
+`bb -f {{path/to/script.clj}}`
+
+- Bind input to a sequence of lines from stdin:
+
+`printf "first\nsecond" | bb -i "(map clojure.string/capitalize *input*)"`
+
+- Bind input to a sequence of EDN values from stdin:
```

Comment:
What is END values?

Reply:
extensible data notation. It's the data format used to write clojure: https://github.com/edn-format/edn

Reply:
> extensible data notation. It's the data format used to write clojure: https://github.com/edn-format/edn

How about adding it too in the command description?

```suggestion
- Bind input to a sequence of EDN(Extensible Data Notation) values from stdin:
```